### DISCUSSION TITLE
GlowTTS + MelGan Inference Runtime Error

### DISCUSSION THREAD

**Discussion by michaellin99999** [coqui-ai/TTS#1093] [General Q&A]
Hello
We tried connecting GlowTTS with Trained MelGan and ran inference and get this error.  cant call numpy() on tensor requires grad.

how do I fix this?

![messageImage_1641819789737](https://user-images.githubusercontent.com/57974973/148770415-6ff7384f-762e-4ced-a052-957409ebe74c.jpg)

**Comment by viewee-ohm** [comment]
use waveform.detach().numpy() instead
